tests/e2e/hibernation_test.go

```diff
@@ -107,7 +107,8 @@ var _ = Describe("Cluster Hibernation with plugin", func() {
 		verifySummaryInHibernationStatus := func(clusterName string, message hibernateSatusMessage) {
 			statusOut := getHibernationStatusInJSON(namespace, clusterName)
 			actualStatus := statusOut[string(summaryInStatus)].(map[string]interface{})["status"].(string)
-			Expect(strings.Contains(string(message), actualStatus)).Should(BeEquivalentTo(true), statusOut)
+			Expect(strings.Contains(string(message), actualStatus)).Should(BeEquivalentTo(true),
+				actualStatus+"\\not-contained-in\\"+string(message))
```

Comment:
ALSO fixes a Panic in the hibernate test when the `statusOut` provided as context to Ginkgo cannot be cast to string